src/LaunchDarkly.EventSource/LaunchDarkly.EventSource.csproj

```diff
@@ -18,7 +18,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="LaunchDarkly.Logging" Version="[1.0.1,3.0.0)" />
+    <PackageReference Include="LaunchDarkly.Logging" Version="[2.0.0,3.0.0)" />
```

Comment:
Because .Net will install the oldest compatible version it is most important to have the minimum be the same for all of then. This will help people who have problems with the binding redirects.